hazelcast/src/main/java/com/hazelcast/internal/diagnostics/NetworkingImbalancePlugin.java

```diff
@@ -30,31 +30,32 @@
 import static java.util.concurrent.TimeUnit.SECONDS;
 
 /**
- * The {@link NetworkingPlugin} is an experimental plugin meant for detecting imbalance in the io system. This plugin will
- * probably mostly be used for internal purposes to get a better understanding of imbalances. Normally imbalances are taken
- * care of by the IOBalancer; but we need to make sure it makes the right choice.
+ * The {@link NetworkingImbalancePlugin} is an experimental plugin meant for detecting imbalance in the io system. This
```

Comment:
Plugin has been renamed since it only deals with imbalance issue. No compatibility needs to be provided on this since it is a 3.9 addition.